tools/merge-tap.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import sys
+
+if __name__ == '__main__':
+  total = 0
+  output = ''
+  for filename in sys.argv[2:]:
+    head, rest = open(filename).read().split('\n', 1)
+    if head.startswith('TAP version'):
+      head, rest = rest.split('\n', 1)
+    if head.startswith('1..'):
+      total += int(head[3:])
+      output += rest
```

Comment:
Will this work without renumbering the individual tests in the files being merged? 
As it stands I think we'll end up with multiple `ok 1 ...`.